src/lib/types.ts

```diff
@@ -247,4 +247,4 @@ export type UIKitOptions = PartialDeep<{
   openChannel: SBUConfig['openChannel']['channel'];
 }>;
 
-export type SendbirdChatInitParams = SendbirdChatParams<Module[]>;
+export type SendbirdChatInitParams = Omit<SendbirdChatParams<Module[]>, 'appId'>;
```

Comment:
👍

Reply:
should be same in https://github.com/sendbird/sendbird-uikit-react/blob/main/scripts/index_d_ts#L116